### PR TITLE
Scenario settings - new attribute

### PR DESCRIPTION
**Pull request by fud4ly**
New attribute to container component.
Added feature to be able to open only one container.

pls apruv

### CODE REVIEW

**Comment by samotion2** [comment]
pozor na naming conventions v store

File: src/store/store.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ export const topology = writable({
   "node_containers": [],
 })
 export const containers = writable([])
+export const visibleContainer = writable("wtf")
```
Comment (samotion2) [inline src/store/store.js:15]: tu

**Review by samotion2 — CHANGES_REQUESTED** [review]
pozor na naming scheme v store

**Review by samotion2 — APPROVED** [review]
nis

File: src/store/store.js (hunk 1 above)
Comment (fud4ly) [reply src/store/store.js]:
```suggestion
export const visibleContainer = writable("")
```
maybe set variable to empty string, the value was set this way for debugging reasons

**Pull request merged**